cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -145,8 +145,14 @@ func (cmd *command) Run() error {
 	cmd.duration = time.Since(start)
 
 	// import certificate
-	if err := cmd.importCertificate(); err != nil {
-		return err
+	qImportCertsStep := cmd.NewStep("Install Kyma certificate locally")
```

Comment:
It would make sense to move this step inside of `cmd.importCertificate()` and print manual import instructions in case the user choses to not automatically do it.